site/src/pages/CreateTemplateGalleryPage/StarterTemplates.tsx

```diff
@@ -23,10 +23,17 @@ const selectTags = (starterTemplatesByTag: StarterTemplatesByTag) => {
 };
 
 const sortVisibleTemplates = (templates: TemplateExample[]) => {
-	// The docker template should be the first template in the list,
-	// as it's the easiest way to get started with Coder.
+	// The tasks-docker template should be first, as it's the easiest way to
+	// get started with Coder. The docker template should be second.
+	const tasksDockerTemplateId = "tasks-docker";
 	const dockerTemplateId = "docker";
 	return [...templates].sort((a, b) => {
+		if (a.id === tasksDockerTemplateId) {
+			return -1;
+		}
+		if (b.id === tasksDockerTemplateId) {
+			return 1;
+		}
```

Comment:
this is technically an unstable sort because you're not handling `a.id === b.id`. there shouldn't ever be duplicates, but it's worth thinking about the ramifications.